app/grandchallenge/algorithms/templates/algorithms/algorithmimage_form.html

```diff
@@ -3,7 +3,7 @@
 {% load url %}
 
 {% block title %}
-    Create An Algorithm Container Image - {{ block.super }}
+    Create Image - {{ algorithm.title }} - {{ block.super }}
```

Comment:
Isn't this template used on update as well?

Reply:
Yes indeed, when I discussed it with James, he said to follow how it is implemented in the breadcrumbs of the page, even if they are not accurate. For this one, the breadcrumbs has a fixed "Create image" for the page even though it should be dynamic create/update. That is why I left it as is.